### ISSUE TITLE
Display shell command when `-p` is given. 

### ISSUE THREAD

**Issue by daynefiler** [snakemake/snakemake-wrappers#78]
snakemake --version
    5.10.0

Not sure if this belongs here, or on the general snakemake page, but it would be really nice to see the shell command for the snakemake wrapper rules when using the `-p` parameter to submit the job.

**Comment by fgvieira** [comment]
Yes, it would be very nice, and have actually an open issue on [snakemake](https://github.com/snakemake/snakemake/issues/641).
